src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingParameterResolver.cs

```diff
@@ -74,28 +75,36 @@ protected string GetFormatOrNull(string value)
             return null;
         }
 
-        // This is an alias for 'sys.randguid'
-        private class RandGuidResolver : BindingParameterResolver
+        // This is an alias for 'sys.guid'
+        private class GuidResolver : BindingParameterResolver
```

Comment:
Well, we could say that you can't use "guid" but I think that would be inconsistent. Definitely going forward we won't add any new ones here. This instance is just special because I'm adding a new alias for an existing feature.